examples/website/i3s/components/tool-bar.js

```diff
@@ -33,46 +43,161 @@ const Title = styled.h3`
     text-align: center;
     padding: 0;
     margin: 0;
+    @media (min-width: 768px) {
+        display: none;
+    }
 `
 
 const ToolContainer = styled.div`
     display: inline-block;
     flex-grow: 1;
-    width: calc(100% * (1/4) - 10px - 1px);
+    width: calc(100% * (1/6) - 10px - 1px);
```

Comment:
Why do we need so difficult calculations? Maybe just do positioning using flex box in parent container?